client/block.c

```diff
@@ -95,13 +105,21 @@ static pthread_mutex_t block_mutex;
 //TODO: this variable duplicates existing global variable g_is_pool. Probably should be removed
 static int g_light_mode = 0;
 static uint32_t cache_bounded_counter = 0;
+static struct orphan_block **orphan_hashtable = NULL;
```

Comment:
When I was reviewing this PR, I checked again and again about this global variable. I suggest name it as a global variable.

Reply:
Every block related variable in block.c isn't named as global

Reply:
Is it hashtable? Looks like array

Reply:
it is an hashtable where buckets are lists

Reply:
@jonano614 global variable original xdag logic is that, g_ variable are visible in all xdag not just in one file, by this logic those variable shouldn't have g_. Do you still want g_ on these variables ?